src/bin/ion/commands/beta/generate/templates/rust/anonymous_type.templ

```diff
@@ -0,0 +1,135 @@
+{# following macro defines an anonymous type as children class for its parent type definition #}
```

Comment:
(🗺️ PR tour) This file adds a new macros for generating anonymous types. This macro generates a new child class for given anonymous type based on its properties (e.g. target_kind_name, abstract_data_type).